README.md

```diff
@@ -12,9 +12,9 @@ A simple electronic money cli app which has the following features:
 - Gradle
 - Docker, Docker Compose
 
-## Setting up
+## Set up
 ### Scalar DB server
-- In this sample we will use Cassandra as storage for our Scalar DB server. Cassandra will also run along with Scalar DB server in containers. The setting for our Scalar DB server is [`database.properties`](./database.properties)
+- In this sample, we will use Cassandra as storage for Scalar DB server. The configuration of Scalar DB server is shown below. (It is also stored in [database.properties](./database.properties))
```

Comment:
As I said, bullet points should not be used for this kind of explanation.
(It should be used to list items/actions in the same level)
Please fix other parts as well.